lib/cartopy/crs.py

```diff
@@ -146,6 +146,31 @@ def domain(self):
             domain = self._domain = sgeom.Polygon(self.boundary)
         return domain
 
+    def _repr_html_(self):
+        """
+        Make a visual representation of the projection and return it as an
+        html element.
+
+        """
+        # Imports.
+        import base64
+        from io import BytesIO
+        import matplotlib.pyplot as plt
+        # Produce a visual repr of the Projection instance.
+        ax = plt.axes(projection=self)
```

Comment:
`fig, ax = plt.subplots(subplot_kw={'projection': self})` to avoid working on the current figure.